problem_builder/templates/html/data_export.html

```diff
@@ -1,12 +1,40 @@
-<h3>Data Export</h3>
+{% load i18n %}
+<h3>{% trans "Data Export" %}</h3>
 
-<p>You can export all student answers to multiple-choice questions to a CSV file here.</p>
+<p>{% trans "You can export all student answers to multiple-choice questions and long-form answers to a CSV file here." %}</p>
 
 <div class="data-export-status"></div>
 
 <div class="data-export-actions">
-  <button class="data-export-download">Download result</button>
-  <button class="data-export-start">Start a new export</button>
-  <button class="data-export-cancel">Cancel current export</button>
-  <button class="data-export-delete">Delete result</button>
+  <button class="data-export-download">{% trans "Download result" %}</button>
+  <button class="data-export-cancel">{% trans "Cancel current export" %}</button>
+  <button class="data-export-delete">{% trans "Delete result" %}</button>
 </div>
+
+<div class="export-options">
+  <div>
+    <label>
+      {% trans "Select which blocks to filter by (selecting none will grab all types):" %}
```

Comment:
Nit: "Filter" is a bit technical; I would suggest "Select which type of problems to include"

You could also consider starting with all types initially selected, which may be less confusing than saying that selecting none is the same as selecting all.